modules/lmdbbackend/lmdbbackend.cc

```diff
@@ -59,6 +59,568 @@ static bool s_first = true;
 static int s_shards = 0;
 static std::mutex s_lmdbStartupLock;
 
+std::pair<uint32_t, uint32_t> LMDBBackend::getSchemaVersionAndShards(std::string& filename)
+{
+  // cerr << "getting schema version for path " << filename << endl;
+
+  uint32_t schemaversion;
+
+  int rc;
+  MDB_env* env = nullptr;
+  MDB_dbi dbi;
```

Comment:
I prefer to declare vars "as late as possible" so they can either be inited at the spot or in the very next statement

Reply:
makes sense!